backend/main.py

```
@@ -451,22 +414,18 @@ async def catch_all(full_path: str):
 # --- Protected Admin Endpoints ---
 from backend.routers.auth import get_current_user
 
 
 @app.post("/admin/seed-database")
 def manual_seed_database(current_user=Depends(get_current_user)):
     """Manual endpoint to seed database (admin only)."""
-    if getattr(current_user, "role", None) != "admin":
-        from fastapi import HTTPException
-
+    if os.getenv("ENVIRONMENT") == "production":
+        raise HTTPException(status_code=404, detail="Not Found")
```

Ruff (0.15.11)

[error] Undefined name `HTTPException`

(F821)